packages/textfield/index.js

```diff
@@ -52,20 +49,19 @@ class WarpTextField extends WarpElement {
   }
 
   get _inputStyles() {
-    return fclasses({
-      [input.default]: true,
-      [input.invalid]: this.invalid,
-      [input.disabled]: this.disabled,
-      [input.readOnly]: this.readOnly,
+    return classes({
+      [input.base]: true,
+      [input.default]: !this.invalid && !this.disabled && !this.readOnly,
+      [input.invalid]: this.invalid && !this.disabled && !this.readOnly,
+      [input.disabled]: !this.invalid && this.disabled && !this.readOnly,
+      [input.readOnly]: !this.invalid && !this.disabled && this.readOnly,
       [input.suffix]: this._hasSuffix,
-      // we style input with prefix here because we cannot use
-      // arbitrary values with commas in UnoCSS like pl-[var(--w-prefix-width, 40px)]
-      'warp-input-with-prefix': this._hasPrefix,
+      [input.prefix]: this._hasPrefix,
```

Comment:
Right now the prefix ends up right on top of the placeholder/input value: 
<img width="242" alt="Skärmavbild 2024-05-06 kl  16 01 59" src="https://github.com/warp-ds/elements/assets/144792260/d15e6330-2c46-414b-99cc-9e92952560e4">
<img width="235" alt="Skärmavbild 2024-05-06 kl  16 02 04" src="https://github.com/warp-ds/elements/assets/144792260/514bbe2f-844b-4073-bff1-5fac3fe2a8fd">

Reply:
Sounds like you might be missing the`@unocss-placeholder` in textfield component's static styles 🤔

Reply:
But is that something I should add myself? Isn't that something that should already be in the textfield component in this branch? 🤔

Reply:
I can understand how this can be confusing! In this package, we are relying on WarpElement's styles, which are always coming from Eik (https://assets.finn.no/pkg/@warp-ds/css/v1). Until the updated styles are released from @warp-ds/css and the v1 alias is updated in Eik, the only way to test our css changes in this repo is by generating styles with UnoCSS locally. For that to work inside Shadow DOM, we need to add a temporary `@unocss-placeholder` to the static styles of a given web component, so UnoCSS understands where to inject the generated CSS. It was explained in our README.md in the section ["How to test component-classes locally"](https://github.com/warp-ds/elements?tab=readme-ov-file#how-to-test-component-classes-locally) 😊

Reply:
Ah right! I forget this all the time! 😄 🙈

Reply:
Yeah now it works like it should! 😄 👍